openedx/core/djangoapps/credentials/utils.py

```diff
@@ -64,3 +64,34 @@ def get_user_program_credentials(user):
         programs_credentials_data = get_programs_for_credentials(user, programs_credentials)
 
     return programs_credentials_data
+
+
+def get_xseries_credentials(user):
```

Comment:
Are there tests for this method? They should be moved as well, or added if they don't exist.